.devcontainer/Dockerfile

```diff
@@ -0,0 +1,19 @@
+FROM mcr.microsoft.com/devcontainers/python:3.11-bullseye
+
+RUN apt-get update && \
+    apt-get install -y python3-venv git curl && \
+    rm -rf /var/lib/apt/lists/*
+
+# Copy the devcontainer folder first
+COPY . /workspaces/py-launch-blueprint-main/.devcontainer
```

Comment:
@Tatianah3 why is it `-main` is there a reason for that? It seems inconsistent.

This should check out to the branch you're on, so it is confusing to have a reference to Main.